### PR TITLE
Updated FAQ content format and links

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updated FAQ content format with recent information at the top of the page. Fixed the broken links in Billing section.

### CODE REVIEW

File: docker-hub/billing/faq.md [hunk 1]
```diff
@@ -93,26 +83,26 @@ When you add seats to your plan in the middle of your billing cycle, a prorated
...
 
 ### How do I downgrade from a Team plan to a Free Team plan?
 
-Before you downgrade to a Free plan, ensure that your organization details are updated to reflect features available in the Free plan. For example, you may need to reduce the number of team members and convert any private repositories to public repositories. For information on what’s included in the Free plan, [see the billing](https://docs.docker.com/docker-hub/billing/){:target="blank" rel="noopener" class=""} page.
+Before you downgrade to a Free plan, ensure that your organization details are updated to reflect features available in the Free plan. For example, you may need to reduce the number of team members and convert any private repositories to public repositories. For information on what’s included in the Free plan, see [the billing](downgrade.md){:target="blank" rel="noopener" class=""} page.
```
Comment (usha-mandya) [inline docker-hub/billing/faq.md:97]: @jerae-duffin Not sure this is a typo, but I believe we should be pointing users to the Pricing page for info on what's included in the Free plan. This section points to the Downgrade page instead.

File: docker-hub/billing/faq.md [hunk 2]
```diff
@@ -93,26 +83,26 @@ When you add seats to your plan in the middle of your billing cycle, a prorated
...
 
 ### How do I downgrade from Pro to a Free plan?
 
-Before you downgrade to a Free plan, ensure that your account organization details are updated to reflect features available in the Free plan. For example, you may need to convert any private repositories to public repositories. For information on what’s included in the Free plan, see [the billing](https://docs.docker.com/docker-hub/billing/){:target="blank" rel="noopener" class=""} page.
+Before you downgrade to a Free plan, ensure that your account organization details are updated to reflect features available in the Free plan. For example, you may need to convert any private repositories to public repositories. For information on what’s included in the Free plan, see [the billing](downgrade.md){:target="blank" rel="noopener" class=""} page.
 
 ### How do I add a member to a team in my organization?
 
-For information on how to add a member to a team, see [Add a member to a team](https://docs.docker.com/docker-hub/orgs/#add-a-member-to-a-team){:target="blank" rel="noopener" class=""}.
+For information on how to add a member to a team, see [Add a member to a team](upgrade.md){:target="blank" rel="noopener" class=""}.
```
Comment (usha-mandya) [inline docker-hub/billing/faq.md:105]: Should this go to https://docs.docker.com/docker-hub/orgs/#add-a-member-to-a-team instead?

**Comment by jerae-duffin** [comment]
@usha-mandya  updated the correct links

**Review by usha-mandya — APPROVED** [review]
LGTM

**Review by mkvitnitsky — APPROVED** [review]

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**